.github/workflows/scripts/build-ispc.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash -e
 echo PATH=$PATH
-cmake -B build -DISPC_PREPARE_PACKAGE=ON -DISPC_INCLUDE_BENCHMARKS=ON
+cmake -B build -DISPC_PREPARE_PACKAGE=ON -DISPC_INCLUDE_BENCHMARKS=ON -DCMAKE_CXX_FLAGS=-Werror
```

Comment:
Why do we add -DCMAKE_CXX_FLAGS=-Werror to github actions but do not add to appveyor and travis builds?

Reply:
Good point. I'll add them. I kinda forgot that we have anything besides Github Actions :)